ray-curator/tests/stages/io/writer/conftest.py

```diff
@@ -0,0 +1,71 @@
+"""Shared fixtures for writer tests."""
+
+import pandas as pd
+import pytest
+
+from ray_curator.tasks import DocumentBatch
+from ray_curator.utils.performance_utils import StagePerfStats
+
+
+@pytest.fixture
+def pandas_document_batch() -> DocumentBatch:
+    """Fixture providing a pandas DataFrame for testing."""
+    df = pd.DataFrame(
+        {
+            "text": ["Hello pandas", "This is pandas test", "Writing with pandas"],
+            "id": ["pd1", "pd2", "pd3"],
+            "category": ["greeting", "test", "demo"],
+            "score": [0.9, 0.8, 0.95],
+        }
+    )
+    return DocumentBatch(
+        task_id="test_pandas_batch",
+        dataset_name="test_dataset",
+        data=df,
+        _metadata={"dummy_key": "dummy_value"},
+        _stage_perf=[
+            StagePerfStats(
+                stage_name="test_stage",
+                process_time=0.1,  # 100ms = 0.1s
+                actor_idle_time=0.0,
+                input_data_size_mb=0.001,  # Small test data
+                num_items_processed=3,  # 3 rows in the DataFrame
+            )
```

Comment:
Do we need to populate this with dummy stats?

Reply:
WDYM? or just keep empty?

Reply:
Does every fixture/test stage need to add dummy stats like the example here? Mainly thinking if we can avoid needing to add this for most tests/fixtures manually

Reply:
I had a similar thought not for perfstats but `pandas_document_batch / pyarrow_document_batch` as I believe that's a relatively common fixture that might live across multiple tests, when we get there we could abstract this out too?